db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveOvirtUseApiFromAttr < ActiveRecord::Migration[6.0]
+  def change
+    compute_resources = ComputeResource.all.unscoped.where(type: 'Foreman::Model::Ovirt')
+    compute_resources.each do |ovirt_resource|
+      ovirt_resource.attrs = ovirt_resource.attrs.except(:ovirt_use_v4)
+      ovirt_resource.save!(validate: false)
+    rescue => e
+      logger.warn("could save compute attributes due to:#{e}")
```

Comment:
This is the only important comment, others are just informative suggestions:
```suggestion
      logger.warn("could not migrate compute attributes for #{ovirt_resource.name} compute resource due to: #{e}")
```

Reply:
done